IBF-Typhoon-model/lib_r/estimate_landfall.R

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env Rscript
```

Comment:
I am not convinced we need a separate function for this. the script is already in make_map.r 
if we want to make a separate script then i suggest to do this in the python pipeline. this is because we plan to move everything to python in next version of the data pipeline 
I will add the 5 line code  in the pipeline.py to prepare this information